src/libexpr/primops/fetchTree.cc

```diff
@@ -152,6 +162,10 @@ static void fetchTree(
             fetchers::Attrs attrs;
             attrs.emplace("type", "git");
             attrs.emplace("url", fixGitURL(url));
+            if (!attrs.contains("exportIgnore")) {
+                // FIXME: exportIgnore := !submodules
```

Comment:
This has not been implemented, yet the tests pass. Not sure why. Consider forbidding submodules + exportIgnore for now. `fetchGit` does not need it.

Reply:
Decided to disallow.